docs/resources/network_zone_record.md

```diff
@@ -62,7 +62,7 @@ The `entry` block supports:
 
 * `value` - **Required** - Entry value.
 
-* `ttl` - *Optional* - Entry time to live (TTL).
+* `ttl` - *Optional* - Entry time to live (TTL) in seconds. Defaults to 300 seconds.
```

Comment:
If not specified, a record's TTL would normally come from the zone's default TTL. As such, I think it'd be best to not provide a default value.

Reply:
Thanks for pointing that out. I can fix that, however, I was following the docs: https://documentation.ubuntu.com/lxd/en/latest/howto/network_zones/#add-or-remove-entries 

> You can use the --ttl flag to set a custom time-to-live (in seconds) for the entry. Otherwise, the default of 300 seconds is used.

How is the network zone default TTL defined?

Reply:
LXD doesn't seem to set a zone default but a record default of 300: https://github.com/canonical/lxd/blob/main/lxd/network/zone/record.go#L206-L208